packages/query/src/block-processor.ts

```diff
@@ -35,6 +52,14 @@ const blankTxSource = new CommitmentSource({
   source: { case: 'transaction', value: { id: new Uint8Array() } },
 });
 
+const getExchangeRateFromValidatorInfoResponse = getValidatorInfo
+  .pipe(getRateData)
+  .pipe(getValidatorExchangeRate);
+
+const getIdentityKeyFromValidatorInfoResponse = getValidatorInfo.pipe(
+  getIdentityKeyFromValidatorInfo,
+);
```

Comment:
These seem like such specific use cases that I figured they'd be better here, rather than in the `getters` package. Thoughts?

Reply:
I'd be in favor of moving them to getters. If we are successful, lots of folks will be wanting to use & contribute to getters.